src/Controller/UserController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace App\Controller;
+
+use App\Model\UserManager;
+use App\Model\TownManager;
+
+/**
+ * Class UserController
+ *
+ */
+class UserController extends AbstractController
+{
+
+    public function add()
+    {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $userManager = new UserManager();
+            $user = [
+                'firstname' => $_POST['firstname'],
+                'lastname' => $_POST['lastname'],
+                'phone' => $_POST['phone'],
+                'street' => $_POST['street'],
+                'townId' => $_POST['townId'],
+                'email' => $_POST['email']
+            ];
+            $userManager->insert($user);
+            header('Location:/home/index');
```

Comment:
You should stop the controller after a redirection header. May be return an empty string